example/src/main.rs

```diff
@@ -1,19 +1,20 @@
 extern crate jack;
+use jack::JackClient;
 use std::io;
 use std::sync::mpsc::{channel, Sender, Receiver};
 use std::str::FromStr;
 
 pub struct SinWave {
     frame_t: f64,
     frequency: f64,
-    out_port: jack::Port,
+    out_port: jack::Port<jack::Owned<jack::Output<jack::Audio>>>,
```

Comment:
@wmedrano This type looks a little gross, but it's really just `Port<Owned<Output<Audio>>>`, which I think is reasonable, especially since inference allows it to be omitted most places and it's easy to create an alias. It's also pretty minimally descriptive, IMO. If we really want to make consumer's lives easier, we could export aliases like `type AudioOutputPort = Port<Owned<Output<Audio>>>` What do you think?